API/API/Repositories/Classes/ProductRepository.cs

```diff
@@ -0,0 +1,83 @@
+﻿
+using API.Context.Context;
+using API.General.Classes.Configure;
+using API.Models;
+using API.Repositories.Interfaces;
+using Microsoft.EntityFrameworkCore;
+
+// <summary>
+// Developer....: Karla Ramos Benitez       USER ID: XKRB
+// </summary>
+namespace API.Repositories.Classes;
+
+/// <summary>
+/// To manage product database data
+/// </summary>
+public class ProductRepository : IProductRepository
+{
+    /// <summary>
+    /// To query product's tables
+    /// </summary>
+    private readonly ProductContext _productContext;
+
+    /// <summary>
+    /// Parameters are passed via dependency injection to query tables
+    /// </summary>
+    public ProductRepository(ProductContext productContext) => _productContext = productContext;
+
+    /// <summary>
+    /// Get product
+    /// </summary>
+    /// <param name="product">get product</param>¿
+    /// <returns> Product </returns>
+    public async Task<ProductModel> GetProduct(int product) => await _productContext.Products.FindAsync(product);
+
+    /// <summary>
+    /// Create product
+    /// </summary>
+    /// <param name="product">create product</param>
+    /// <returns>create product</returns>
+    public async Task<ProductModel> CreateProduct(ProductModel product)
+    {
+        _ = _productContext.Products.Add(product);
+        _ = await _productContext.SaveChangesAsync();
+        return product;
+    }
+
+    /// <summary>
+    /// Update product
+    /// </summary>
+    /// <param name="product">product updated</param>
```

Comment:
You have to be more clear with the comments. Imagine that someone else is going to read. Do you thing he will understand without context?

Reply:
Done